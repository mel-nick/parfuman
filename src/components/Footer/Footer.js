import React from 'react';
import {fade, withStyles, withTheme, makeStyles} from '@material-ui/core/styles';
import {display, flexbox} from '@material-ui/system';
import TextField from '@material-ui/core/TextField';
import Box from '@material-ui/core/Box';
import MenuList from '../MenuList/MenuList'
import SocialNetworksList from "../SocialNetworksList/SocialNetworksList";
import Logo from "../Logo/Logo";


const useStyles = makeStyles((theme) => ({
        footer: {
            display: 'flex',
            justifyContent: 'space-between',
            alignItems: 'center',
            maxHeight: '120px',
            padding: '10px 20px',
            fontFamily: 'Roboto, sans-serif',
            backgroundColor: theme.palette.primary.dark,
            borderTop: '1px solid' + theme.palette.secondary.dark

        },
        footerSignWrap: {
            display: 'flex',
            flexDirection: 'column',
            textAlign: 'right',
            color: theme.palette.secondary.light
        },
        footerYear: {
            fontSize: '12px'
        },
        footerPhone: {
            fontSize: '12px'
        },
        footerEmail: {
            fontSize: '12px'
        },
    }));

export default function Footer() {
    const classes = useStyles();
    return (
        <footer className={classes.footer} id='footer'>
            <Logo footer={true}/>
            <SocialNetworksList/>
            <p className={classes.footerSignWrap}>
                <span className={classes.footerPhone}>0-800-777-00-00</span>
                <span className={classes.footerEmail}>team@parfume.com</span>
                <span className={classes.footerYear}>Â© 2019</span>
            </p>
        </footer>
    );
}