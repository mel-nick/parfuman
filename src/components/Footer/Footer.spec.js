import React from 'react';
import toJson from 'enzyme-to-json';
import Footer from './Footer';

describe('Footer component', () => {
    it('Footer component render', () => {
        const wrapper  = shallow(<Footer />)
        expect(toJson(wrapper)).toMatchSnapshot()
    })
})