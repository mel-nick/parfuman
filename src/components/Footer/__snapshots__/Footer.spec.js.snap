// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Footer component render 1`] = `
<footer
  className="makeStyles-footer-1"
  id="footer"
>
  <Logo
    footer={true}
  />
  <SocialNetworksList />
  <p
    className="makeStyles-footerSignWrap-2"
  >
    <span
      className="makeStyles-footerPhone-4"
    >
      0-800-777-00-00
    </span>
    <span
      className="makeStyles-footerEmail-5"
    >
      team@parfume.com
    </span>
    <span
      className="makeStyles-footerYear-3"
    >
      Â© 2019
    </span>
  </p>
</footer>
`;
